hail/python/hail/ir/ir.py

```diff
@@ -919,32 +919,34 @@ def _compute_type(self, env, agg_env):
 
 
 class AggArrayPerElement(IR):
-    @typecheck_method(array=IR, name=str, agg_ir=IR, is_scan=bool)
-    def __init__(self, array, name, agg_ir, is_scan):
+    @typecheck_method(array=IR, element_name=str, index_name=str, agg_ir=IR, is_scan=bool)
+    def __init__(self, array, element_name, index_name, agg_ir, is_scan):
```

Comment:
Can you describe what `index_name` represents just to make sure what I think it is (index into aggregator array) is correct?

Reply:
(described in answer to the other comment)